content/en/docs/setup/independent/troubleshooting-kubeadm.md

```diff
@@ -280,4 +280,20 @@ but this will result in the key `enable-admission-plugins` only having the value
 A known workaround is to use the kubeadm
 [configuration file](/docs/setup/independent/control-plane-flags/#apiserver-flags).
 
+## The NodeRegistration.Taints field is omitted when marshalling kubeadm configuration
+
+*Note: This [issue](https://github.com/kubernetes/kubeadm/issues/1358) only applies to tools that marshal kubeadm types (e.g. to a YAML configuration file). It will be fixed in kubeadm API v1beta2.*
+
+By default, kubeadm applies the `role.kubernetes.io/master:NoSchedule` taint to master nodes.
```

Comment:
we should use `control-plane node` instead of `master nodes` in the user text.
(even if the label itself is actually master...)

Reply:
Will fix, thanks!

Reply:
Fixed